sensor/NetworkStatsSensor.js

```diff
@@ -48,28 +53,137 @@ class NetworkStatsSensor extends Sensor {
     this.processPingConfigure()
     this.pingResults = {}
     this.checkNetworkPings = {}
+    this.sampleJobs = {}
+    this.processJobs = {}
+  }
+
+  sleep(seconds) {
```

Comment:
there is always a delay util function, check it out

Reply:
fixed with "delay"